src/borg/fuse.py

```diff
@@ -259,10 +260,17 @@ def _create_filesystem(self):
                     self.pending_archives[archive_inode] = archive.name
 
     def get_item(self, inode):
+        item = self._read_cache.get(inode)
+        if item:
+            return item        
         try:
-            return self._items[inode]
+            item = self._items[inode]
+            self._read_cache[inode] = item
```

Comment:
this is not needed, it is "caching" just a pure-dictionary get operation.

this is rather counterproductive as a dict-get is fast and caching it takes away a LRU cache entry that could be used better.